README.md

```diff
@@ -146,36 +148,19 @@ database. If your MDX doesn't reference other files (or only imports things from
 
 ## Compilation target
 
-We use rollup to bundle your MDX and its dependencies and babel to compile
-things. We're using `@babel/preset-env` and you can configure this using the
-standard browserlist configuration.
-[learn more](https://babeljs.io/docs/en/babel-preset-env#browserslist-integration).
+We use [esbuild](https://esbuild.github.io/) to bundle your MDX and its
+dependencies. esbuild handles compiling
```

Comment:
Is there any way to control what version of JavaScript it's compiled down to? Like which browsers to support?

Reply:
whoops, I didn't finish that sentance.